### DISCUSSION TITLE
Asking for Experiments Results

### DISCUSSION THREAD

**Discussion by YangHangWu** [CMU-SAFARI/Hermes#9] [General]
Hi all,
It's great to know this project and the github instructions are in great detailed.
We are working on research and trying to replicate their results; however, the experiments take forever for us to get the result.
I'm wondering could you kindly share the CSV files `rollup.csv` and `rollup_mcpat.csvwith` us so that we can start to analyze the results.

It's will be really helpful if you can sent those files to my email: rsps10517@gmail.com or whatever way you feel comfortable. 
Thank you.

**Comment by rahulbera** [comment]
The excel file provided in the experiment directory (https://github.com/CMU-SAFARI/Hermes/blob/main/experiments/MICRO22_AE.xlsx) contains all the stats from our runs. You can use it for replicating the results. I’ll update it with the power stats soon.

**Comment by rahulbera** [comment]
@YangHangWu The power number is now added to the MICRO22_AE.xlsx file.

Closing this issue for now. We can reopen it if required.
